node/src/components/consensus.rs

```diff
@@ -313,6 +314,13 @@ where
             }
             Event::Shutdown => handling_es.shutdown_if_necessary(),
             Event::FinishedJoining(timestamp) => handling_es.finished_joining(timestamp),
+            Event::ConsensusRequest(requests::ConsensusRequest::IsBondedValidator(
+                era_id,
+                pk,
+                responder,
+            )) => responder
+                .respond(handling_es.is_bonded_validator(era_id, pk))
+                .ignore(),
```

Comment:
The other `HandlingEraSupervisor` methods return effects, so maybe this should, too?
```rust
)) => handling_es.is_bonded_validator(era_id, pk, responder),
```

Reply:
Do you want me to add a method that will be just this 1 line?

Reply:
It's not important; I just thought that it would be more important with the other `HandlingEraSupervisor` methods.
(To clarify, I'd replace `is_bonded_validator` with a method that returns the effect.)